### DISCUSSION TITLE
Session management of outputs for users

### DISCUSSION THREAD

**Discussion by MazarSayed** [langfuse/langfuse#4478] [Support] [answered]
### Describe the feature or potential improvement

Ability to store outputs and load them back through langfuse, since we are already storing the traces, user_id, input and outputs. It would be an easy feature to introduce session management for storing outputs and loading them back through langfuse.

**Comment by marcklingen** [comment, marked as answer]
Hi @MazarSayed, if you want to, you can fetch all traces by a specific user via the sdks/api. Have a look here: 

These endpoints also include filters by `userId`.
